### ISSUE TITLE
Background color of selected file is not obvious enough to identify

### ISSUE THREAD

**Issue by iseeltf** [samdark/intellij-visual-studio-code-dark-plus#63]
As the picture shows：
<img width="461" alt="image" src="https://github.com/samdark/intellij-visual-studio-code-dark-plus/assets/7991777/c9b8a0e2-fec0-4468-89fe-2e2f4034e968">
IntelliJ IDEA 2023.3.3 (Ultimate Edition)
Build #IU-233.14015.106, built on January 25, 2024

**Comment by n1kk** [comment]
I agree, I also struggle to identify it in the tree without bringing focus there. Can't always be blue? Or at least have an option for it?

**Comment by samdark** [comment]
Duplicate of https://github.com/samdark/intellij-visual-studio-code-dark-plus/issues/58
